scripts/install_third_party_libs.py

```diff
@@ -125,9 +125,25 @@ def pip_install(package, version, install_path):
 
     # The call to python -m is used to ensure that Python and Pip versions are
     # compatible.
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', '%s==%s' % (package, version),
-        '--target', install_path])
+    command = [
+        sys.executable, '-m', 'pip', 'install', '%s==%s'
+        % (package, version), '--target', install_path]
+    process = subprocess.Popen(
+        command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = process.communicate()
+    if not stderr or 'pip install --upgrade pip' in stderr:
```

Comment:
I saw that error comes because we use pip 9. version, but it isnt an error I think, its probably a warning as the packages still get installed, and when there are other install errors, that message does not come.

Reply:
Oh, I get it, some commands use stderr as a user output. But this is not really good approach towards this, you should instead check if the `returncode` of the `process` is zero, that should mean that the process finished successfully.

Reply:
Done! Sorry it took long I didn't expect backend tests to fail after that commit, Ill try fix the backend tests asap.